packages/components/psammead-media-player/src/Placeholder/index.jsx

```diff
@@ -13,6 +13,14 @@ const StyledPlaceholder = styled.div`
   left: 0;
   width: 100%;
   height: 100%;
+
+  ${({ noJsClassName }) =>
+    noJsClassName &&
+    css`
+      .${noJsClassName} & {
```

Comment:
Do you need this `.${noJsClassName} & {`? I had a similar doubt this week https://github.com/bbc/psammead/pull/2785#discussion_r355326357

Reply:
Have updated.